src/components/components/ebay-notice-base/index.marko

```diff
@@ -12,13 +13,14 @@ static var ignoredAttributes = [
     "title",
     "footer",
     "type",
-    "mainRoot"
+    "mainRoot",
+    "noA11yLabel"
 ];
 
 $ var status = input.status;
 $ var prefixClass = input.prefixClass;
 <${input.root || "section"}
-    aria-labelledby:scoped="status"
+    aria-labelledby=(!input.noA11yLabel && component.elId('status'))
     class=[prefixClass, input.class]
     ...processHtmlAttributes(input, ignoredAttributes)>
     <${input.headerRoot || "div"} class=`${prefixClass}__header` id:scoped="status"
```

Comment:
Can you check and make sure that this value is the same in the normal case. From what I remember it might be different.
If it's different, have it be set by `component.elId('status')`

Reply:
Page notice and Section Notice still look good and have the aria-labelledby

<img width="500" alt="Screen Shot 2021-05-28 at 3 38 23 PM" src="https://user-images.githubusercontent.com/25092249/120048145-30078e80-bfcb-11eb-8707-0c1cc679bf26.png">
<img width="487" alt="Screen Shot 2021-05-28 at 3 38 50 PM" src="https://user-images.githubusercontent.com/25092249/120048147-3138bb80-bfcb-11eb-9344-5ee937b78e70.png">